packages/tasks_panel/src/utils/ws-folder.ts

```diff
@@ -7,3 +7,15 @@ export function cleanTasks(tasks: ConfiguredTask[]): void {
     delete _.index;
   });
 }
+
+export const DEPLOY = "Deploy";
+export const BUILD = "Build";
+export const MISC = "Miscellaneous";
+
+export function isMatchDeploy(intent: string): boolean {
+  return /^deploy.*$/gi.test(intent);
```

Comment:
is it OK to check if it exactly starts with "build"/"deploy"? or is it enough that it includes it somewhere?

Reply:
this is the exact definition